server/docker/server.Dockerfile

```diff
@@ -18,6 +18,9 @@ RUN add-apt-repository \
 RUN apt-get -y update --fix-missing
 RUN apt-get -y install docker-ce
 
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install tzdata
+RUN apt-get -y install postgresql-10
```

Comment:
Why postgres 10 instead of 12?

Reply:
It's the latest that's available in the ubuntu bionic repos. Let me look around to see what the recommended way of installing a later version is.